x/sequencers/keeper/msg_server.go

```diff
@@ -113,3 +118,97 @@ func (m msgServer) UpsertSequencer(goCtx context.Context, msg *types.ConsensusMs
 
 	return &types.ConsensusMsgUpsertSequencerResponse{}, nil
 }
+
+// defines the list of accounts we want to bump the sequence
+var handleAccounts = map[string]struct{}{
+	proto.MessageName(&authtypes.BaseAccount{}):                 {},
+	proto.MessageName(&vestingtypes.BaseVestingAccount{}):       {},
+	proto.MessageName(&vestingtypes.ContinuousVestingAccount{}): {},
+	proto.MessageName(&vestingtypes.DelayedVestingAccount{}):    {},
+	proto.MessageName(&vestingtypes.PeriodicVestingAccount{}):   {},
+	proto.MessageName(&vestingtypes.PermanentLockedAccount{}):   {},
+}
+
+const BumpSequence = 10_000_000_000
+
+func (m msgServer) BumpAccountSequences(goCtx context.Context, msg *types.MsgBumpAccountSequences) (*types.MsgBumpAccountSequencesResponse, error) {
+	if msg.Authority != m.authority {
+		return nil, sdkerrors.ErrorInvalidSigner.Wrapf("only an authorized actor can bump account sequences")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	var allErrors error
+	m.accountKeeper.IterateAccounts(ctx, func(account authtypes.AccountI) bool {
```

Comment:
why u not breaking with error on the first error?

Reply:
fixed